python/ray/experimental/state/api.py

```diff
@@ -658,6 +824,31 @@ def list_jobs(
     raise_on_missing_output: bool = True,
     _explain: bool = False,
 ):
+    """List jobs submitted in the cluster by the ray job tool.
```

Comment:
```suggestion
    """List jobs submitted to the cluster by the ray job submission.
```

Also add a reference link?